go/cmd/vttlstest/vttlstest.go

```diff
@@ -60,49 +58,58 @@ func init() {
 		"CreateSignedCert":     cmdCreateSignedCert,
 		"RevokeCert":           cmdRevokeCert,
 	}
-}
 
-var (
-	root = flag.String("root", ".", "root directory for certificates and keys")
-)
+	servenv.OnParseFor("vttlstest", func(fs *flag.FlagSet) {
+		fs.StringVar(&root, "root", root, "root directory for certificates and keys")
+	})
+}
 
 func cmdCreateCA(subFlags *flag.FlagSet, args []string) {
-	subFlags.Parse(args)
+	if err := subFlags.Parse(args); err != nil {
+		log.Fatalf("Error parsing flags: %v", err)
+	}
 	if subFlags.NArg() > 0 {
 		log.Fatalf("CreateCA command doesn't take any parameter")
 	}
 
-	tlstest.CreateCA(*root)
+	tlstest.CreateCA(root)
 }
 
 func cmdCreateCRL(subFlags *flag.FlagSet, args []string) {
-	subFlags.Parse(args)
+	if err := subFlags.Parse(args); err != nil {
+		log.Fatalf("error parsing flags: %v", err)
+	}
+
 	if subFlags.NArg() != 1 {
 		log.Fatalf("CreateCRL command takes a single CA name as a parameter")
 	}
 
 	ca := subFlags.Arg(0)
-	tlstest.CreateCRL(*root, ca)
+	tlstest.CreateCRL(root, ca)
 }
 
 func cmdRevokeCert(subFlags *flag.FlagSet, args []string) {
 	parent := subFlags.String("parent", "ca", "Parent cert name to use. Use 'ca' for the toplevel CA.")
 
-	subFlags.Parse(args)
+	if err := subFlags.Parse(args); err != nil {
+		log.Fatalf("Error parsing flags: %v", err)
+	}
```

Comment:
all these checks are unnecessary, because the sub-command flag set will [`ExitOnError`](https://github.com/vitessio/vitess/pull/11361/files#diff-d939da5e2ba6926fd1051722b09b0ad1229f9088645b459ce7258ecf6cf04db9L155)